web/app/models.py

```diff
@@ -48,6 +49,15 @@ class User(db.Model, UserMixin):
     login_count = db.Column(db.Integer)
     players = relationship("Player")
 
+    # For every AGA member, a fake user (aga_id, claimed=False) is created.
+    # (This fake user holds onto their real life game history for them.)
+    # When that AGA member signs up, the fake user is updated to 
+    # (aga_id, claimed=True), and the real user gets (aga_id, claimed=False).
+    # Thus, only one fake and one real user can ever exist for an aga_id.
```

Comment:
Correct, the current DB uses whatever number the TD reports, so players have games under both ids.  Theres a table mapping the ~25 odd people with vanity ids.